src/ServiceControl/Notifications/Email/EmailSender.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ServiceControl.Notifications.Email
+{
+    using System;
+    using System.Net;
+    using System.Net.Mail;
+    using System.Threading.Tasks;
+    using NServiceBus.Logging;
+
+    class EmailSender
+    {
+        public static async Task Send(EmailNotifications settings, string subject, string body, string emailDropFolder = null)
+        {
+            try
+            {
+                using (var client = CreateSmtpClient(settings, emailDropFolder))
+                {
+                    using (var mailMessage = new MailMessage(settings.From, settings.To, subject, body))
+                    {
+                        await client.SendMailAsync(mailMessage).ConfigureAwait(false);
+                    }
+                }
+            }
+            catch (Exception e)
+            {
+                log.Warn("Failure sending email.", e);
+                throw;
+            }
+        }
+
+        static SmtpClient CreateSmtpClient(EmailNotifications settings, string emailDropFolder)
+        {
+            if (emailDropFolder != null)
+            {
+                return new SmtpClient
```

Comment:
What do you think about this?

https://docs.microsoft.com/en-us/dotnet/api/system.net.mail.smtpclient?redirectedfrom=MSDN&view=net-5.0#remarks

![image](https://user-images.githubusercontent.com/174258/116521947-e563fb00-a8d4-11eb-8826-b33ea510411d.png)

Reply:
@tmasternak raised it in the beginning but I called yagni on this based on the github link from that warning https://github.com/dotnet/platform-compat/blob/master/docs/DE0005.md. It says it is OK to use it for occasional emails from tooling.